simba/costs.py

```diff
@@ -718,7 +728,10 @@ def to_csv_lists(self):
                     row.append(num)
             output.append(row)
 
-        transposed_output = []
-        for column, _ in enumerate(output[0]):
-            transposed_output.append([row[column] for row in output])
-        return transposed_output
+        if transpose:
+            transposed_output = []
+            for column, _ in enumerate(output[0]):
+                transposed_output.append([row[column] for row in output])
+            output = transposed_output
+
```

Comment:
remove this and optional argument. keeping the transpose code if its not used is not valuable.